packages/chart/src/plotly/plotly-extends.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Fix some issues with plotly types.
+ */
+
+declare module 'plotly.js' {
```

Comment:
Can this go in the root `@types` folder? I'd try putting it there to see if you get the same results. It is meant to serve as a central types override

Reply:
It doesn't seem to merge with the existing Plotly types when I do this